models/yang/sonic/sonic-neighbors.yang

```diff
@@ -0,0 +1,155 @@
+module sonic-neighbors {
+    namespace "http://github.com/Azure/sonic-neighbors";
+    prefix snbrs;
+    yang-version 1.1;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import sonic-port {
+        prefix prt;
+    }
+
+    import sonic-portchannel {
+        prefix spc;
+    }
+
+    import sonic-vlan {
+        prefix svlan;
+    }
+
+    import sonic-mgmt-port {
+        prefix smt;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC yang for NEIGH_TABLE";
+
+    revision 2019-10-10 {
+        description
+            "Initial revision.";
+    }
+
+    rpc clear_neighbors {
+        input {
+            leaf force {
+                type boolean;
+                description "Set it to True to delete PERMANENT/Static entries";
+            }
+
+            leaf family {
+                type enumeration {
+                    enum IPv4;
+                    enum IPv6;
+                }
+                description "Address family (IPv4/IPv6)";
+            }
+
+            choice option {
+                leaf all {
+                    type boolean;
+                    description
+                        "Parameter for clearing all dynamic entries from ARP table";
+                }
+
+                leaf ip {
+                    type inet:ip-prefix;
+                    description
+                        "Parameter for deleting ARP entries related to the given IPv4 address";
```

Comment:
Only IPv4 address?

Reply:
Good catch, I implemented IPv4 first and missed to update the description for IPv6.  I have updated the description.